example/ion-sfu-gstreamer-send/main.go

```diff
@@ -0,0 +1,93 @@
+package main
+
+import (
+	"flag"
+	gst "github.com/pion/ion-sdk-go/pkg/gstreamer-src"
+
+	ilog "github.com/pion/ion-log"
+	sdk "github.com/pion/ion-sdk-go"
+	"github.com/pion/webrtc/v3"
+)
+
+var (
+	log = ilog.NewLoggerWithFields(ilog.DebugLevel, "", nil)
+)
+
+func main() {
+	// parse flag
+	var session, addr string
+	flag.StringVar(&addr, "addr", "localhost:50051", "Ion-sfu grpc addr")
+	flag.StringVar(&session, "session", "test room", "join session name")
+	audioSrc := flag.String("audio-src", "audiotestsrc", "GStreamer audio src")
+	videoSrc := flag.String("video-src", "videotestsrc", "GStreamer video src")
+	flag.Parse()
+
+	// add stun servers
+	webrtcCfg := webrtc.Configuration{
+		ICEServers: []webrtc.ICEServer{
+			webrtc.ICEServer{
+				URLs: []string{"stun:stun.stunprotocol.org:3478", "stun:stun.l.google.com:19302"},
+			},
+		},
+	}
+
+	config := sdk.Config{
+		WebRTC: sdk.WebRTCTransportConfig{
+			Configuration: webrtcCfg,
+		},
+	}
+	// new sdk engine
+	e := sdk.NewEngine(config)
+
+	// get a client from engine
+	c, err := sdk.NewClient(e, addr, "client id")
+
+	var peerConnection *webrtc.PeerConnection = c.GetPubTransport().GetPeerConnection()
+
+	peerConnection.OnICEConnectionStateChange(func(state webrtc.ICEConnectionState) {
```

Comment:
instead of this i this we should use this

```
t, _ := c.GetPubTransport().GetPeerConnection().AddTransceiverFromTrack(producer.AudioTrack(), webrtc.RTPTransceiverInit{
						Direction: webrtc.RTPTransceiverDirectionSendonly,
					})

					defer c.UnPublish(t)
					t2, _ := c.GetPubTransport().GetPeerConnection().AddTransceiverFromTrack(producer.VideoTrack(), webrtc.RTPTransceiverInit{
						Direction: webrtc.RTPTransceiverDirectionSendonly,
					})
					defer c.UnPublish(t2)
c.OnNegotiationNeeded()
```

Reply:
this is the same code as used in Publish()